### PR TITLE
test

### PR DESCRIPTION
**Pull request by O3Ol**
test

### CODE REVIEW

**Comment by PsiACE** [comment]
pls rm simple-sqlite-host-call/sqlite_host.out & test.db

File: simple-sqlite-host-call/doc.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#
```
Comment (PsiACE) [inline simple-sqlite-host-call/doc.md:1]: add a title?

File: simple-sqlite-host-call/doc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+wat2wasm sqlite_host.wat -o sqlite_host.wasm
```
Comment (PsiACE) [inline simple-sqlite-host-call/doc:1]: doc or doc.md? just need one of them

File: simple-sqlite-host-call/README.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
+# Example
```
Comment (PsiACE) [inline simple-sqlite-host-call/README.md:1]: format it ?

File: simple-sqlite-host-call/doc (hunk 2 above)
Comment (O3Ol) [reply simple-sqlite-host-call/doc:1]: in readme.md

File: simple-sqlite-host-call/README.md (hunk 3 above)
Comment (O3Ol) [reply simple-sqlite-host-call/README.md:1]: how?

File: simple-sqlite-host-call/doc (hunk 2 above)
Comment (O3Ol) [reply simple-sqlite-host-call/doc:1]: This one is which I will delete.

File: simple-sqlite-host-call/doc.md (hunk 1 above)
Comment (O3Ol) [reply simple-sqlite-host-call/doc.md:1]: This is a test doc...

**Pull request closed without merging**